### PR TITLE
fix(highlights): missing highlights when accessing static methods and referencing methods

### LINKED ISSUE #94: Bug: Missing `@type` hightlight
Missing highlight when:
 -  accessing static methods
 -  referencing methods 
 -  type parameters
        
image
image

### PR DESCRIPTION
**Pull request by captainko**
Fix #94 

Checklist:
- [ ] All tests pass in CI.
- [ ] There are sufficient tests for the new fix/feature.
- [x] Grammar rules have not been renamed unless absolutely necessary.
- [x] The conflicts section hasn't grown too much.
- [x] The parser size hasn't grown too much (check the value of STATE_COUNT in src/parser.c).

### CODE REVIEW

**Review by aryx — APPROVED** [review]

**Comment by captainko** [comment]
@p00f Please check this out.

File: queries/highlights.scm [hunk 1]
```diff
@@ -23,13 +23,21 @@
   name: (identifier) @type)
 (enum_declaration
   name: (identifier) @type)
+(type_parameter
+  (identifier) @type)
```
Comment (maxbrunsfeld) [inline queries/highlights.scm:27]: I'm ok with this for now, but I think it'd be even better to change the grammar to alias `identifier` as a `type_identifier` inside of the `type_parameter` rule; then we wouldn't need this. If you're interested, we could do that as part of this PR, but we could also merge as-is.

Comment (captainko) [reply queries/highlights.scm]: @maxbrunsfeld, Thanks for the suggestion. I have alias `identifier` to `type_indentifier`. Please take a look.

**Comment by maxbrunsfeld** [comment]
@dcreager @rewinfrey @patrickt Is there any chance that any of you folks at GitHub could look into why GitHub actions doesn't seem to reliably run for this repository? I wonder if it's a bug in Actions, or if we just don't have it set up  properly. It looks to me like the actions workflow is configured properly, but we just never get a status report back for any PRs, so it's hard to merge PRs confidently.

**Comment by rewinfrey** [comment]
Apologies for the bump on this PR @maxbrunsfeld and @captainko,  but pulling in the updates to the workflow file appears to have kicked off CI.

@maxbrunsfeld in case you continue seeing this in future PR's,  please feel free to ping us! 🙇‍♂️

**Comment by maxbrunsfeld** [comment]
Thanks so much for getting that working @rewinfrey ❤️.

**Comment by maxbrunsfeld** [comment]
Thanks for the PR @captainko.

**Pull request merged**